manifests/init.pp

```diff
@@ -92,7 +92,7 @@
         }
       }
       default: {
-        fail ("gitlab_ci_runner::manage_repo parameter for ${::osfamily} is not supported.")
+        fail("gitlab_ci_runner::manage_repo parameter for ${::osfamily} is not supported.")
```

Comment:
This change does not look related to this PR.
If this line is changed we needs to call the fact via $facts and remove leading `::`.
IMO this should be in an other PR.

Reply:
Yes, you are right. Thanks. I fixed it too.

Reply:
It is interesting case. Test will be fail. I returned ${::osfamily}
I recommend to use $facts['os']['family']
It is decision by author.

Reply:
It is strange when i run tests locally with `fail("gitlab_ci_runner::manage_repo parameter for ${facts['os']['family']} is not supported.")` the command ` bundle exec rake test` does not fail.

What was the problem you encountered ?